workspace/src/common/launch/art_launch/package.xml

```diff
@@ -13,7 +13,6 @@
   <exec_depend>art_localization_launch</exec_depend>
   <exec_depend>art_control_launch</exec_depend>
   <exec_depend>art_planning_launch</exec_depend>
-  <exec_depend>art_sensing_launch</exec_depend>
```

Comment:
why remove this?

Reply:
oops didn't mean to push that. It was bothering me bcs it kept giving warnings bcs non of the dependencies for it are built. I don't think you should be required to build this when in simulation, how do you feel about having a different metapackage for building in sim vs reality?

Reply:
`art_sensing_launch` doesn't link to any packages, so I don't think you need to worry about that, right?